plugins/vpp/l3plugin/vppcalls/vpp1908/route_dump_test.go

```diff
@@ -35,13 +35,40 @@ func TestDumpStaticRoutes(t *testing.T) {
 	ifIndexes.Put("if1", &ifaceidx.IfaceMetadata{SwIfIndex: 1})
 	ifIndexes.Put("if2", &ifaceidx.IfaceMetadata{SwIfIndex: 2})
 
-	ctx.MockVpp.MockReply(&ip.IPFibDetails{
-		Path: []ip.FibPath{{SwIfIndex: 2}},
-	})
-	ctx.MockVpp.MockReply(&vpe.ControlPingReply{})
-	ctx.MockVpp.MockReply(&ip.IP6FibDetails{
-		Path: []ip.FibPath{{SwIfIndex: 1}},
-	})
+	ctx.MockVpp.MockReply(&ip.IPRouteDetails{
+		Route: ip.IPRoute{
+			Prefix: ip.Prefix{
+				Address: ip.Address{
+					Af: ip.ADDRESS_IP4,
+					Un: ip.AddressUnion{
```

Comment:
Please use generated constructors [`AddressUnionIP4`](https://github.com/ligato/vpp-agent/blob/dev/plugins/vpp/binapi/vpp1908/ip/ip.ba.go#L289) and [`AddressUnionIP6`](https://github.com/ligato/vpp-agent/blob/dev/plugins/vpp/binapi/vpp1908/ip/ip.ba.go#L306) here.